### PR TITLE
Forward combined post publication messages into Kafka

### PR DESCRIPTION
**Pull request by BerniVarga**
Service responsibilities:
* Consumes msgs from PostPublicationEvents
* Consumes msgs from PostMetadataPublicationEvents
* Builds combined messages (by calling the document-store-api, or public-annotations-api for the missing parts of the combined content)
* Forwards msgs to the CombinedPostPublicationEvents queue

Current status: It follows to be tested in XP.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+		go msgProcessor.ProcessMessages()
+
+		// route admin requests
+		routeRequests(port, NewCombinerHealthcheck(*kafkaProxyAddress, *kafkaProxyRoutingHeader, &client, *contentTopic, *metadataTopic, *combinedTopic, *docStoreApiBaseURL, *publicAnnotationsApiBaseURL))
+	}
+
+	logrus.SetLevel(logrus.InfoLevel)
+	logrus.Infof("PostPublicationCombiner is starting with args %v", os.Args)
+
+	err := app.Run(os.Args)
```
Comment (tamas-molnar) [inline main.go]: Can you log the os.args or the app config at startup please?

File: processor/data_combiner.go [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+	if status == http.StatusNotFound {
+		return c, nil
+	}
+
+	if err != nil {
+		return c, err
+	}
+
+	if err := json.Unmarshal(b, &c); err != nil {
+		return c, errors.New(fmt.Sprintf("Could not unmarshall content with uuid=%v, error=%v", uuid, err.Error()))
```
Comment (tamas-molnar) [inline processor/data_combiner.go]: You can replace `errors.New(fmt.Sprintf(...))` with `fmt.Errorf(...)` everywhere.

File: processor/msg_processor.go [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+	//parse message - collect data, then forward it to the next queue
+	var cm model.MessageContent
+	b := []byte(m.Body)
+	if err := json.Unmarshal(b, &cm); err != nil {
+		logrus.Errorf("Could not unmarshall message with TID=%v, error=%v", tid, err.Error())
+		return
+	}
+
+	// wordpress, brightcove, methode-article - the system origin is not enough to help us filtering. Filter by contentUri.
+	if supports(p.config.SupportedContentURIs, cm.ContentURI) {
```
Comment (tamas-molnar) [inline processor/msg_processor.go]: Maybe invert the condition, so you get rid of a level of indentation? Same in `processMetadataMsg`.

File: main.go (hunk 1 above)
Comment (BerniVarga) [inline main.go]: os.Args are logged above

**Pull request merged**